### PR TITLE
Added more functionalities to "Folder as workspace"

### PR DESCRIPTION
**Pull request by SinghRajenM**
Added more functionalities to "Folder as workspace"
1. Fold All
2. UnFold All

### CODE REVIEW

**Comment by donho** [comment]
Please merge several PR in one to make code review easier.

File: PowerEditor/src/NppCommands.cpp [hunk 1]
```diff
@@ -111,6 +111,8 @@ void Notepad_plus::command(int id)
 					if (_pFileBrowser->isClosed())
 					{
 						_pFileBrowser->display();
+						checkMenuItem(IDM_VIEW_FILEBROWSER, true);
```
Comment (SinghRajenM) [inline PowerEditor/src/NppCommands.cpp]: This is part of PR #2263

Comment (SinghRajenM) [inline PowerEditor/src/NppCommands.cpp]: This is part for PR #2263

Comment (SinghRajenM) [inline PowerEditor/src/NppCommands.cpp]: This is part of PR #2263

Comment (SinghRajenM) [inline PowerEditor/src/NppCommands.cpp]: This is part for PR #2263

Comment (SinghRajenM) [inline PowerEditor/src/NppCommands.cpp]: This is part of PR #2263

**Comment by SinghRajenM** [comment]
@donho It is single PR except above part. Rest all five files changes belong to single PR.
However, the above part is also related to this change. Do you want to separate it out?

File: PowerEditor/src/WinControls/FileBrowser/fileBrowser_rc.h [hunk 2]
```diff
@@ -31,24 +31,24 @@
 
 #define	IDD_FILEBROWSER		3500
 
-#define IDD_FILEBROWSER_MENU			 (IDD_FILEBROWSER + 10)
+#define IDD_FILEBROWSER_MENU				(IDD_FILEBROWSER + 10)
```
Comment (donho) [inline PowerEditor/src/WinControls/FileBrowser/fileBrowser_rc.h]: All the reformating of code will be rejected. Check coding standard:
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md

**Comment by donho** [comment]
1. I don't think this feature is useful for a lot of users.
2. 1 COMMIT PER PR!

**Pull request closed without merging**